rpc/cloudagent/compute/galleryimage/moc_cloudagent_galleryimage.proto

```diff
@@ -59,6 +59,8 @@ message GalleryImage {
 	string sourcePath = 11;
 	Tags tags = 12;
 	ImageSource sourceType = 13;
+	string version = 14;
+	CloudInitDataSource cloudInitDataSource = 16;
```

Comment:
I dont see this in the design md

Reply:
the version is in vm-download.md and the cloudInitDataSource is in the vm-provision.md file